### PR TITLE
TPROD 312/TPROD 351 - CoreBundle Helper templates cleanup

### PR DESCRIPTION
**Pull request by LordRembo**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:
This PR is to remove the unused CoreBundle Helper php templates, and replace references to these Helper files in other places. There is also some minor overlap with [this cleanup PR](https://github.com/mautic/mautic/pull/12032/)
Some of these changes include:
- removal of some MauticSocialBundle templates that were previously reverted, including index file (haven't seen issues)
- removal of some Standard templates

#### Steps to test this PR:
- Since the Helper files are used just about everywhere, probably all pages need to be checked.
- some of the helpers are responsible for pagination, modals, lists, tables, buttons, tabs, …

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.

### CODE REVIEW

**Review by escopecz — APPROVED** [review]
The code changes make sense. Thanks! 👍

File: app/bundles/CoreBundle/Views/Standard/list.html.php [hunk 1]
```diff
@@ -1,205 +0,0 @@
...
-/*
- * @copyright   2014 Mautic Contributors. All rights reserved
- * @author      Mautic
- *
- * @link        http://mautic.org
- *
- * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
- */
-
-if ('index' == $tmpl) {
```
Comment (escopecz) [inline app/bundles/CoreBundle/Views/Standard/list.html.php:12]: Although removing this file without its twig counterpart will result with the same error mentioned in https://github.com/mautic/mautic/pull/12021

**Review by escopecz — CHANGES_REQUESTED** [review]
We'll need the standard list.html.twig. Ensure that the Marketing Messages UI is working.

File: app/bundles/CoreBundle/Views/Standard/list.html.php (hunk 1 above)
Comment (LordRembo) [reply app/bundles/CoreBundle/Views/Standard/list.html.php:12]: > We'll need the standard list.html.twig. Ensure that the Marketing Messages UI is working.

That file already exists (and is called by the Controller). I haven't seen any issues on the Marketing Messages UI.

**Comment by LordRembo** [comment]
As mentioned on the change request: the Standard list.html.twig file already exists (and is called by the Controller) so the Marketing Messages should be alright. Haven't seen any issues myself but I'm also not sure what the original issue was so anyone willing to double-check would be appreciated.

**Comment by sdoering** [comment]
On the Contact Detail View I discovered this in the drop down menu:
image

When trying to click on the DNC SMS User ("Henry Catalano") I encounter an 500 server error:
image

From the log file: 
```
[2023-03-01 13:03:10] mautic.CRITICAL: Uncaught PHP Exception Twig\Error\RuntimeError: "Impossible to access an attribute ("timestamp") on a null variable." at /var/www/html/app/bundles/PageBundle/Views/SubscribedEvents/Timeline/index.html.twig line 5 {"exception":"[object] (Twig\\Error\\RuntimeError(code: 0): Impossible to access an attribute (\"timestamp\") on a null variable. at /var/www/html/app/bundles/PageBundle/Views/SubscribedEvents/Timeline/index.html.twig:5)"} {"hostname":"mautic-web","pid":2284}
```

In the segment view when clicking on the drop down in front of the segment name I encounter this padding/margin error:
image

The same happens in the respective dropdown in the emails list: 
image


Having clicked the switch to unpublish a Report I am not able to click it again to publish it again. Only after a reload of the page, I can toggle the switch again to publish. 

image

When opening the test email and opening the builder I can't close the builder again (buttons have no effect):
image

**Comment by mabumusa1** [comment]
Maybe not related to this PR, but we used to have a picture for Anonymous contacts 

image

**Comment by mabumusa1** [comment]
Delete is not aligned with the other icons
image

The translation is missing for Delete
image

Bulk Actions 
image

**Comment by LordRembo** [comment]
Interesting! May indeed not be related to the cleanup but probably straightforward to fix. Will take a look in the morning (GMT+1).

File: app/bundles/CoreBundle/Views/Standard/list.html.php (hunk 1 above)
Comment (escopecz) [reply app/bundles/CoreBundle/Views/Standard/list.html.php:12]: I missed that. Thanks for checking!

**Comment by LordRembo** [comment]
@sdoering @mabumusa1 Can you confirm if the mentioned bugs are now solved? I copied fixes from https://github.com/mautic/mautic/pull/12046/

**Review by mabumusa1 — APPROVED** [review]
Based on the files that are changed in this PR I did the following tests: 

- [X] Views of the base and slim pages worked as expected 
- [X] Actions worked
- [X] Bulk actions
- [X] Charts with data 
- [X] Empty Charts
- [X] Modals
- [X] Delete buttons
- [X] Pagination 
- [X] Search results 
- [X] Maps
- [X] Theme selection
- [X] Marketplace 
- [X] No results in search 
- [X] Tables 
- [X] Tabs
- [X] Filters 
- [X] Reports
- [X] Contact timeline

I think there are places in the `SocialBundle` and monitoring that I could not test.

**Comment by RCheesley** [comment]
@LordRembo is there anything that we can do to address the drop in code coverage on this PR? Appreciate that we are removing stuff so it might be expected.

**Comment by LordRembo** [comment]
@RCheesley I'm not sure how relevant that drop is for this PR, since there's no direct impact on functionality (nothing was added or changed except bugfixes). 
(EDIT: also some updates in an Extension to match what was originally in a php template)
I can see a lot is coming from the ['indirect changes'](https://app.codecov.io/gh/mautic/mautic/pull/12039/indirect-changes) but those functions should already have been in use before this PR. Maybe there just needs to be separate tickets for increasing test coverage on functionality already in use?

**Comment by LordRembo** [comment]
For example, there's not much CoreBundle Helper stuff that has tests at the moment, like only a handful of the Extensions and php functions. That could be its own issue to tackle.

**Review by escopecz — APPROVED** [review]
The code changes look good. Thanks! 👍

**Pull request merged**